tealer/teal/instructions/instructions.py

```diff
@@ -36,6 +37,14 @@ def line(self) -> int:
     def line(self, l: int):
         self._line = l
 
+    @property
+    def comment(self) -> str:
+        return self._comment
+
+    @comment.setter
```

Comment:
I know we want to keep being consistent, so this change is fine, but is there any reason we use a property with no side effect instead of having the comment as a normal field?

cc: @montyly I guess?

Reply:
Not really. The main advantage here is that it will keep consistency, and ease docstring integration

Reply:
I have little prior experience with Python, so I don't have a strong opinion on that.